JavaErickHernandez/Day3/Student.java

```diff
@@ -0,0 +1,37 @@
+package bootcamp;
+public class Student {
+	Integer id;
+	String name;
+	double cgpa;
```

Comment:
cgpa?

Reply:
Thats the name of the variable in the excercise, i will change it